packages/desktop-electron/menu.ts

```diff
@@ -75,64 +93,70 @@ function getMenu(isDev, createWindow, budgetId) {
           role: 'paste',
         },
         {
-          role: 'pasteandmatchstyle',
+          role: 'pasteAndMatchStyle',
         },
         {
           role: 'delete',
         },
         {
-          role: 'selectall',
+          role: 'selectAll',
         },
       ],
     },
     {
       label: 'View',
       submenu: [
-        isDev && {
-          label: 'Reload',
-          accelerator: 'CmdOrCtrl+R',
-          click(item, focusedWindow) {
-            if (focusedWindow) focusedWindow.reload();
-          },
-        },
+        isDev
+          ? {
+              label: 'Reload',
+              accelerator: 'CmdOrCtrl+R',
+              click(_item, focusedWindow) {
+                if (focusedWindow) focusedWindow.reload();
+              },
+            }
+          : { label: 'hidden', visible: false },
```

Comment:
Previously it was putting a stray ```false``` into the array which is an invalid value. I've changed it to a hidden menu item so it respects the types. I think this is easier to read than doing manual push outside of this setup.